web/src/components/snapshots/ConfigureSnapshots.jsx

```diff
@@ -103,10 +103,23 @@ class ConfigureSnapshots extends React.Component {
                         <a href="https://velero.io/docs/v1.6/supported-providers/" target="_blank" rel="noopener noreferrer" className="snapshotOptions">
                           <span style={{ width: "130px" }}>  <span className="icon cloudIcon u-cursor--pointer u-marginRight--5" /> Other provider  </span>
                           <span className="icon external-link-icon u-cursor--pointer u-marginLeft--30" /></a>
-                        <a className="snapshotOptions" onClick={() => openConfigureFileSystemProviderModal(FILE_SYSTEM_NFS_TYPE)}>
-                          <span style={{ width: "130px" }}>  <span className="icon nfsIcon u-cursor--pointer u-marginRight--5" /> NFS </span></a>
-                        <a className="snapshotOptions" onClick={() => openConfigureFileSystemProviderModal(FILE_SYSTEM_HOSTPATH_TYPE)}>
-                          <span style={{ width: "130px" }}>  <span className="icon hostpathIcon u-cursor--pointer u-marginRight--5" /> Host Path </span></a>
+                        { snapshotSettings.isMinioDisabled ?
+                            <>
+                              <a href="https://github.com/replicatedhq/local-volume-provider" target="_blank" rel="noopener noreferrer" className="snapshotOptions">
+                                <span style={{ width: "130px" }}>  <span className="icon nfsIcon u-cursor--pointer u-marginRight--5" /> NFS </span>
+                                <span className="icon external-link-icon u-cursor--pointer u-marginLeft--30" /></a>
+                              <a href="https://github.com/replicatedhq/local-volume-provider" target="_blank" rel="noopener noreferrer" className="snapshotOptions">
+                                <span style={{ width: "130px" }}>  <span className="icon hostpathIcon u-cursor--pointer u-marginRight--5" /> Host Path </span>
+                                <span className="icon external-link-icon u-cursor--pointer u-marginLeft--30" /></a>
+                            </>
+                          :
+                            <>
```

Comment:
whooh, i didn't this was possible

Reply:
You mean the [fragment](https://reactjs.org/docs/fragments.html)? Picking up the occasional React tip :D